rpcd/playbooks/roles/elasticsearch/tasks/upgrade_elasticsearch_pre.yml

```diff
@@ -108,13 +108,13 @@
     - reindex-wrapper
 
 - name: Drop log_test1 template
-  shell: 'curl -fsk -XDELETE {{ internal_lb_vip_address }}:{{ elasticsearch_http_port }}/_template/log_test1'
-  args:
-     warn: no
-  register: curl_result
+  uri:
+    url: "{{ internal_lb_vip_address }}:{{ elasticsearch_http_port }}/_template/log_test1"
+    method: DELETE
```

Comment:
Recommend to add 'return_content: yes', source: http://docs.ansible.com/ansible/uri_module.html